expyfun/_version.py

```diff
@@ -1,2 +1,2 @@
 __version__ = '2.0.0.dev0'
-__fork__ = '/Eric89GXL'
+__fork__ = 'Unknown'
```

Comment:
it really should say `'LABSN'`.  If it says `'Unknown'` then `setup.py` did not detect a `.git` directory, meaning expyfun was probably downloaded as a zip file (or was installed via `download_version`).  If it says anything else, it reflects `git config --get remote.origin.url` which is a really bad idea, since we (the devs) have an unwritten convention of setting the LABSN remote as `upstream` and setting our personal forks as `origin`.  I move that you leave this as-is for now (as `'Unknown'`), and that the `__fork__` machinery be dismantled and removed.  My use case for it was marginally justifiable at the time, and no longer exists.